crates/vault-registry/src/lib.rs

```diff
@@ -1035,7 +1033,7 @@ impl<T: Config> Pallet<T> {
             increase: tokens.amount(),
         });
 
-        Ok((new_to_be_replaced, new_collateral))
+        Ok((new_to_be_replaced, vault.replace_collateral()))
```

Comment:
It doesn't really make sense to me to return the replace collateral in a function that does nothing with the collateral. Maybe just add a getter? Also the comment above the function is outdated